Tasks/JavaToolInstallerV0/FileExtractor/JavaFilesExtractor.ts

```diff
@@ -74,7 +74,7 @@ export class JavaFilesExtractor {
         tar.execSync();
     }
 
-    private extractFiles(file: string, fileEnding: string) {
+    private async extractFiles(file: string, fileEnding: string) {
```

Comment:
async should not be needed here since severZipExtract is not async and unzipExtract call is removed.